packages/dashboard/src/app/views/exploreTemplates/content/templateGridItem.js

```diff
@@ -0,0 +1,144 @@
+import {
+  TemplateDisplayContent,
+  CardWrapper,
+  PicturePoster,
+} from './components';
+
+export const FOCUS_TEMPLATE_CLASS = 'focus_template';
+
+const TemplateGridItem = forwardRef(
+  (
+    {
+      detailLink,
+      handleCreateStory,
+      handleFocus,
+      height,
+      id,
+      isActive,
+      posterSrc,
+      slug,
+      title,
+      status,
+    },
+    ref
+  ) => {
+    const tabIndex = isActive ? 0 : -1;
+    const posterAltText = sprintf(
+      /* translators: %s: Template title. */
+      __('First page of %s template', 'web-stories'),
+      title
+    );
+    return (
+      <CardGridItem
+        ref={ref}
+        id={`template-grid-item-${id}`}
+        className="templateGridItem"
+        data-testid={`template-grid-item-${id}`}
+        $posterHeight={height}
+        onFocus={handleFocus}
+      >
+        <Container>
+          <CardWrapper $isSelected={isActive}>
+            <PicturePoster>
+              <source srcSet={posterSrc.webp} type="image/webp" />
+              <source srcSet={posterSrc.png} type="image/png" />
+              <img src={posterSrc.png} alt={posterAltText} />
+            </PicturePoster>
+            <Gradient />
+            <Scrim
+              data-testid="card-action-container"
+              data-template-slug={slug}
+            >
+              <TemplateDisplayContent>
+                <Button
+                  size={BUTTON_SIZES.SMALL}
+                  type={BUTTON_TYPES.SECONDARY}
+                  as="a"
+                  ariaLabel={sprintf(
+                    /* translators: %s: template title.*/
+                    __('Go to detail view of %s', 'web-stories'),
+                    title
+                  )}
+                  href={detailLink}
+                  disabled={!detailLink}
+                  className={FOCUS_TEMPLATE_CLASS}
+                  tabIndex={tabIndex}
+                >
+                  {TEMPLATES_GALLERY_ITEM_CENTER_ACTION_LABELS[status]}
+                </Button>
+
+                <Button
+                  size={BUTTON_SIZES.SMALL}
+                  type={BUTTON_TYPES.PRIMARY}
+                  onClick={handleCreateStory}
+                  disabled={!handleCreateStory}
+                  tabIndex={tabIndex}
+                  ariaLabel={sprintf(
+                    /* translators: %s: template title.*/
+                    __('Create new story from %s', 'web-stories'),
+                    title
+                  )}
+                >
+                  {__('Use template', 'web-stories')}
+                </Button>
+              </TemplateDisplayContent>
+            </Scrim>
+          </CardWrapper>
+        </Container>
+      </CardGridItem>
+    );
+  }
+);
+
+TemplateGridItem.displayName = 'TemplateGridItem';
+
+TemplateGridItem.propTypes = {
+  detailLink: PropTypes.string,
+  handleCreateStory: PropTypes.func,
+  handleFocus: PropTypes.func.isRequired,
```

Comment:
Nit: I think for props the prefix that is usually used when passing in event handlers is `on` rather than `handle` which is used when creating event handlers.

Could these be updated to be `onCreateStory` and `onFocus`?